test/functional/testlib.bash

```diff
@@ -1537,14 +1537,6 @@ create_third_party_repo() { #swupd_function
 		create_bundle -n os-core -v "$version" -f /usr/lib/os-release:"$OS_RELEASE",/usr/share/clear/update-ca/Swupd_Root.pem:"$cert",/usr/share/defaults/swupd/format:"$FORMAT" -u "$repo_name" "$env_name"
 	fi
 
-	if [ "$TEST_ENV_ONLY" = true ]; then
```

Comment:
why are you removing this part? I find very useful to use the `create_test_environment_only` function from my terminal to create an environment and then debug a scenario manually within that environment. The info printed here is helpful to set up the debugger.

Reply:
This variables doesn't exist anymore. I can review again if there's anything left behind, but I am not opposed to export all variables needed to debug that when create_test_environment_only is used.